### PR TITLE
Classifier - TESS Light Curve Viewer - add Graph2dRangeX task (for Task Area)

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview
package: `lib-classifier`

This PR adds the Graph2dRangeX task to the Classifier's Task Area.

screen shot 2019-01-18 at 14 21 12
_The Graph2dRangeX task can be seen on the right side. Yes, that bit of text. That's all._

This task area is currently a very simple copy of the Drawing Task placeholder, which only has space for `instructions`, and nothing more. The Task (in the Task Area) has _no interactive bits._

There is currently no intention to improve this Task any further (save maybe for some styling updates?), at least until we get a design review on the UX.

### Testing
The Task can be tested on any project workflow with the custom `graph2dRangeX` task. For convenience, try `http://localhost:8080/?project=1862` on staging.

### Status
Ready for review.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
👌 Thanks @rogerhutchings - I've added tests to the Task (component) and the Task (store) which was also missing from a previous update.

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    const { addAnnotation, task } = this.props
+    if (event.target.checked) addAnnotation(index, task)
+  }
+
+  render () {
+    const {
+      annotations,
+      task
+    } = this.props
+    let annotation
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js]: Is this being used in the render method at all?

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Graph2dRangeXTask.wrappedComponent.defaultProps = {
+  addAnnotation: () => {},
+  annotations: observable.map(),
+  task: {}
+}
+
+Graph2dRangeXTask.wrappedComponent.propTypes = {
+  addAnnotation: PropTypes.func,
+  annotations: MobXPropTypes.observableMap,
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js]: I didn't know about this - v cool!

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  return {
+    addAnnotation,
+    annotations
+  }
+}
+
+@inject(storeMapper)
+@observer
+class Graph2dRangeXTask extends React.Component {
+  onChange (index, event) {
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js]: Is this actually being used?

**Comment by shaunanoordin** [comment]
## PR Update
This PR has been updated to add a new marker counter.

screen shot 2019-01-22 at 13 32 47

NOTE: the numbers won't display properly until PR #378 is merged, since the LightCurveViewer component in this branch hardcodes Task `'T100'` as its 'drawing' task.

TODO:
- add translations to the text `marks made`
- (another PR) allow user to click on a button, or something, on the Task component to trigger Annotate mode. This is a UI/UX improvement.

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js (hunk 3 above)
Comment (shaunanoordin) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js]: No - `onChange` has now been removed.

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js (hunk 2 above)
Comment (shaunanoordin) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js]: I actually took this from the `DrawingTask.js` code - I thought you introduced this? 😮

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js (hunk 1 above)
Comment (shaunanoordin) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js]: It should be - there's supposed to be a "X marks made" counter/button on the Task area. I've added it in, in the latest commit in this PR, to make this prop less dangly.

screen shot 2019-01-22 at 13 40 45
_What the Task Area is supposed to ultimately mimic_

**Comment by shaunanoordin** [comment]
## PR Update
- `counterpart` translations have been added to the 'Marks made' text in the Graph2dRangeXTask component.

### Status
Ready for re-review.

**Comment by srallen** [comment]
@rogerhutchings are the changes ok here?

**Pull request merged**